### ISSUE TITLE
Feedbkack Needed: Automatically detect browser and use browser history

### ISSUE THREAD

**Issue by captn3m0** [captn3m0/hackertray#24]
**tl;dr**: Skip to bottom, run the command, and report its output.

Currently, the `--chrome` switch is tedious and requires knowledge of where your chrome config lives. Most people just have a single profile in the browser. So I'm thinking of making it magical and automatically "detect your browser", so as to use its browser history.

So, the issues I'm facing:
1. Detecting default browser reliably across a range of distros.
2. Detecting which profile to use in that browser.

For (1), I'm thinking of going with `xdg-settings`. Its quite common, and should be a portable solution. We currently use python's webbrowser module, which uses `gnome-open` to open a webpage in default browser. `xdg-settings` will give me an easy way out to detect the default browser.

For (2), I think I'll use the default profile, and allow user to change it with a switch if needed.

**HELP NEEDED**

If you're a hackertray user, run the following command, and let me know if it detects your default browser or not, including the case where its not available on your system.

`xdg-settings get default-web-browser`

It runs fine on Debian and Ubuntu in my tests, but I'd love to learn if it works on yours.
